scripts/upgrade

```diff
@@ -27,6 +27,11 @@ nextcloud_domain=$(ynh_app_setting_get --app="$app" --key=nextcloud_domain)
 #=================================================
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=2
 
+if ynh_compare_current_package_version --comparison le --version 6.4.10~ynh2
+then
+	ynh_die "Upgrade from version 6.4.10 is not possible. You must uninstall and reinstall Collabora package manually"
```

Comment:
```suggestion
	ynh_die --message="Upgrade from version 6.4.10 is not possible. You must uninstall and reinstall Collabora package manually"
```

Reply:
Interesting! Indeed, the documentation shows that we need that argument.

It I am not wrong, the positional argument is still permitted by this function, but is legacy. Of course it is really better to avoid legacy syntax, my question is about other projects which have the old signature. Is it relevant that I propose a PR to remove the positional arguments for them?